config/database.yml

```diff
@@ -5,21 +5,22 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  host: localhost
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/development
```

Comment:
As above, @Heidar has suggested we follow the name scheme app_environment here, e.g. laa_great_ideas_development, laa_great_ideas_test, laa_great_ideas_production.